odoo/tools/xml_utils.py

```diff
@@ -83,3 +88,49 @@ def create_xml_node(parent_node, node_name, node_value=None):
     :returns (etree._Element):
     """
     return create_xml_node_chain(parent_node, [node_name], node_value)[0]
+
+
+def load_xsd_from_url(env, url, module_name):
```

Comment:
It would be nice to have an extra parameter to force the refresh of the cached attachment